File: 11734/Main.java
import java.io.*;
import java.util.*;

public class Main {
   public static void main(String args[]) throws IOException {
      StringBuilder answer = new StringBuilder();
      BufferedReader in = new BufferedReader(new InputStreamReader(System.in));
      int n = Integer.parseInt(in.readLine());

      for (int t = 1; t <= n; t++) {
         String team = in.readLine();
         String judge = in.readLine();
         StringBuilder a = new StringBuilder();
         StringBuilder b = new StringBuilder();
         StringTokenizer tokenizer;

         tokenizer = new StringTokenizer(team);
         while (tokenizer.hasMoreTokens()) {
            a.append(tokenizer.nextToken());
         }

         tokenizer = new StringTokenizer(judge);
         while (tokenizer.hasMoreTokens()) {
            b.append(tokenizer.nextToken());
         }

         answer.append("Case ");
         answer.append(Integer.toString(t));
         answer.append(": ");
         if (!a.toString().equals(b.toString())) {
            answer.append("Wrong Answer\n");
         } else {
            answer.append((team.length() == judge.length()) ? "Yes\n" : "Output Format Error\n");
         }
      }
      System.out.print(answer);
   }
}

File: 11734/main.c
#include <stdio.h>

int main(int argc, char* argv[])
{
   int t, n, i, j;
   int team_len, judge_len, space;
   char wrong_answer;
   char input[30];
   char team[30];
   char judge[30];

   scanf("%d\n", &n);
   for (t = 1; t <= n; t++) {
      space = 0;

      gets(input);
      team_len = 0;
      for (i = 0; input[i] != '\0'; i++) {
         if (input[i] == ' ') {
            ++space;
            continue;
         }
         team[team_len++] = input[i];
      }
      team[team_len++] = '\0';

      gets(input);
      judge_len = 0;
      for (i = 0; input[i] != '\0'; i++) {
         if (input[i] == ' ') {
            --space;
            continue;
         }
         judge[judge_len++] = input[i];
      }
      judge[judge_len++] = '\0';

      wrong_answer = 0;
      if (team_len == judge_len) {
         for (i = j = 0; team[i] != '\0' && judge[j] != '\0'; i++, j++) {
            if (team[i] != judge[j]) {
               wrong_answer = 1;
               break;
            }
         }
      } else {
         wrong_answer = 1;
      }

      if (wrong_answer) {
         printf("Case %d: Wrong Answer\n", t);
      } else {
         printf("Case %d: %s\n", t, (space == 0) ? "Yes" : "Output Format Error");
      }
   }
   return 0;
}
